### ISSUE TITLE
Indent/Unindent

### ISSUE THREAD

**Issue by fde101** [prominic/Moonshine-IDE#151]
Seems the biggest missing features in the editor (from my perspective) that I have come across so far while playing with the IDE are options to select tab width, to use spaces instead of tabs, and to indent/unindent selected blocks of code (shift text left or right by the set number of spaces).  A feature to convert tabs to spaces would be a nice plus.  Sorry if I missed them, but I'm not seeing these features anywhere.

**Comment by piotrzarzycki21** [comment]
Hi @fde101 ,

First of all Thank you for trying Moonshine and that feedback! It's precious to us. We will definitely look into that!

Piotr

**Comment by piotrzarzycki21** [comment]
@fde101 ,

I would like to understand whether we are on the same page. Are you searching following settings for code editor ? 
![image](https://user-images.githubusercontent.com/24554795/35513931-e24573a8-0504-11e8-8ff2-397c96639ff9.png)

By this one:
> selected blocks of code (shift text left or right by the set number of spaces).

Do you mean something like that when you use Shift + Tab and Tab only ?
![tab_indent](https://user-images.githubusercontent.com/24554795/35513986-1a067490-0505-11e8-83bb-99d9c6266b26.gif)

**Comment by fde101** [comment]
Yes... thanks, I couldn't find the tab or shift+tab thing in the menus and had no idea what the shortcuts were.  As to the first one, I still can't find those options.
I also just noticed the the options to hide the application / hide others are missing from the application menu under Mac OS X.

**Comment by piotrzarzycki21** [comment]
I just provide you screenshot from Intellij, we don't have such options in the configuration yet. 

@rat-moonshine can you provide a bit more information in case of Mac OSX. Should those options be available in the menu ? Is it some kind of standard ? I'm more Windows user. :)

**Comment by fde101** [comment]
Those menu items are described here: https://developer.apple.com/macos/human-interface-guidelines/menus/menu-bar-menus/

The "Hide" option is present in the Dock menu at least, so that helps.

**Comment by rat-moonshine** [comment]
Thank you for the document link @fde101 . As you have noticed, Moonshine menus constructed custom with different options/ordering etc. We can work on including Hide/Un-hide option to the menu, though (as this seems an option which may uses by different users, like you. We were not much used to use this option personally, though, so we never included).

I've created #162 as a separate issue to this feature request.
